### PR TITLE
Rename tmlock test to remove JIRA references

### PR DESCRIPTION
**Pull request by tushar-dadlani**
Currently there are tests in the codebase that have JIRA references,
this commit is an example of a potential refactoring across the codebase.

### CODE REVIEW

File: src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py [hunk 1]
```diff
@@ -1,20 +1,3 @@
-"""
```
Comment (jimmyyih) [inline src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py]: I think this still needs to be here... but I'm not a lawyer so yeah.

File: src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py [hunk 2]
```diff
@@ -23,33 +6,24 @@
...
-class test_mpp19535(MPPTestCase):
+class test_tmlock(MPPTestCase):
 
     @classmethod
     def setUpClass(cls):
         db = Database()
         db.setupDatabase()
         os.environ['PGDATABASE'] = os.environ['USER']
 
-    def test_mpp19535(self):
```
Comment (jimmyyih) [inline src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py]: Removing the JIRA reference is a good thing.  However, it needs to be replaced with a very detailed comment on what the issue was, what was discovered, and what was fixed from the JIRA.  If you could decipher the JIRA comments and summarize it here, that'd be great.  If not, we can help produce a summary for you.

**Comment by cjcjameson** [comment]
Seems like a good idea. Can the intentionality of the test be preserved somehow, perhaps by Pivotal employees putting in a comment with the upshot of the regression, or baking the intentionality into the name or output of the test?

Also, I think we'd keep the copyright in place.

**Comment by cjcjameson** [comment]
Oooh, and also might have to worry about test name collisions? Or is that not an issue...

**Comment by tushar-dadlani** [comment]
@jimmyyih I wiill retain the legal notice and deal with that separately. I tried to understand the contents of the JIRA ticket but wasn't able to capture all the intent. I will take some help with someone from your team to understand it better and then resubmit the PR.

File: src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py (hunk 1 above)
Comment (hlinnaka) [inline src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py]: Well, it's still copyrighted to Pivotal, so we, Pivotal, are free to do whatever we want. We don't have copyright and license notices like this in most files, and certainly wouldn't add one here if we were starting from scratch, so +1 on removing it. Removing the notice won't change the fact that Pivotal holds the copyright and it's Apache licensed - the notices at the top level of the repository cover this, like all other files.

**Comment by ashwinstar** [comment]
@asubramanya is looking into the same test currently and high likelihood the test might get deleted altogether, replaced by a simple assertion in code. So, I think co-ordinating on this would be good with him.

**Comment by tushar-dadlani** [comment]
@asubramanya Do you have any thoughts about this ?

**Comment by asubramanya** [comment]
@tushar-dadlani We have decided to remove the test and add an assert in the code to make sure that the lock is released after DTM recovery. See PR https://github.com/greenplum-db/gpdb/pull/2626
I think this PR can be closed.

File: src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py [hunk 3]
```diff
@@ -23,33 +6,30 @@
...
-        @description Test MPP-19535, TmLock issue
-        @created 2013-04-22 00:00:00
-        @modified 2013-04-22 00:00:00
-        @tags storage DTM schedule_transaction
-        @product_version gpdb: [4.2.5.2- main]
-        """
-
+    def test_tmlock(self):
+        # Summary:
+	# The problem occurs because some backend acquires a lock, does not
```
Comment (d) [inline src/test/tinc/tincrepo/mpp/gpdb/tests/storage/transaction_management/tmlock/test_tmlock.py]: whitespace misaligned

**Comment by Chibin** [comment]
File got removed https://github.com/greenplum-db/gpdb/pull/2626, so closing this.
Feel free to re-open if that's not the case.

**Pull request closed without merging**